testing/regress/ecl-test

```diff
@@ -273,8 +274,10 @@ class RegressMain:
         parser_run.add_argument('--publish', '-p', help="Publish compiled query instead of run.",
                                 action='store_true')
         parser_run.add_argument('--handleEclccWarningFile', '-w', help="Create/overwrite/delete ECLCC warning file.",
-                                action='store_true')
-
+                                action='store_true')                                                   
+        parser_run.add_argument('--cleanup', '-c', choices=['all', 'workunits', 'passed', 'none'], help="Select the cleanup mode: none(default)/all/workunits/passed",
```

Comment:
To force the user to be more mindful I would remove the '-c' parameter choice.

Reply:
yes, will do.